stdlib/src/base64/base64.mojo

```diff
@@ -151,3 +151,85 @@ fn b64decode(str: String) -> String:
 
     p.append(0)
     return p
+
+
+# ===----------------------------------------------------------------------===#
+# b16encode
+# ===----------------------------------------------------------------------===#
+
+
+fn b16encode(str: String) -> String:
+    """Performs base16 encoding on the input string.
+
+    Args:
+      str: The input string.
+
+    Returns:
+      Base16 encoding of the input string.
+    """
+    alias lookup = "0123456789ABCDEF"
+    var b16chars = lookup.data()
+
+    var length = len(str)
+    var out = List[Int8](capacity=length * 2 + 1)
+
+    @parameter
+    @always_inline
+    fn s(idx: Int) -> Int:
```

Comment:
Question to maintainers. I am aware, that this is consistent with the other implementations here, but this name is not meaningful. How about adjusting that?

Reply:
Are you asking about the `b16encode`/`b16decode` names or the local functions (in your snippet, `s`)?

Reply:
done